src/AuthProvider.tsx

```diff
@@ -47,23 +47,24 @@ export interface AuthProviderProps extends UserManagerSettings {
 
     /**
      * On remove user hook. Can be a async function.
-     */
-    onRemoveUser?: () => Promise<void> | void;
-
-    /**
-     * On sign out redirect hook. Can be a async function.
      * Here you can change the url after the logout.
+     *
      * ```jsx
      * const onSignOutRedirect = (): void => {
```

Comment:
This example should be renamed as well just to make it clear that this is a function you'd be able to pass as the `onRemoveUser` prop